packages/suite-native/android/app/build.gradle

```diff
@@ -1,5 +1,9 @@
 apply plugin: "com.android.application"
 
+if (getGradle().getStartParameter().getTaskRequests().toString().contains("firebaseDevelop")){
```

Comment:
@Nodonisko FYI This should ensure that the google service only goes into test builds.